features/cells/src/main/java/com/wire/android/feature/cells/ui/ConversationFilesScreen.kt

```diff
@@ -141,18 +149,33 @@ fun ConversationFilesScreenContent(
     WireScaffold(
         modifier = modifier,
         topBar = {
-            WireCenterAlignedTopAppBar(
-                onNavigationPressed = { navigator.navigateBack() },
-                title = screenTitle ?: stringResource(R.string.conversation_files_title),
-                navigationIconType = navigationIconType,
-                elevation = dimensions().spacing0x,
-                actions = {
-                    MoreOptionIcon(
-                        contentDescription = R.string.content_description_conversation_files_more_button,
-                        onButtonClicked = { optionsBottomSheetState.show() }
-                    )
+            Column {
+                WireCenterAlignedTopAppBar(
+                    onNavigationPressed = { navigator.navigateBack() },
+                    title = screenTitle ?: stringResource(R.string.conversation_files_title),
+                    navigationIconType = navigationIconType,
+                    elevation = dimensions().spacing0x,
+                    actions = {
+                        MoreOptionIcon(
+                            contentDescription = R.string.content_description_conversation_files_more_button,
+                            onButtonClicked = { optionsBottomSheetState.show() }
+                        )
+                    }
+                )
+                breadcrumbs?.let {
+                    LazyRow(
```

Comment:
Maybe we could reverse the breadcrumbs so that they are always scrolled to the end to the current folder and not to the root, that I think would be better for the UX 🤔 
```suggestion
                breadcrumbs?.reversedArray()?.let {
                    LazyRow(
                        reverseLayout = true,
```

Reply:
I will need to check with the design team for this case if we need to show last item on the list or the root